CODE_OF_CONDUCT.md

```diff
@@ -1,36 +1,114 @@
 # Contributor Covenant Code of Conduct
 
 ## Our Pledge
+Diversity is one of the greatest strengths a community can have and many times
+that strength is born from the friction that can only come through sharing of
+differing perspectives.
 
-In the interest of fostering an open and welcoming environment, we as
+In the interest of fostering an open, welcoming, and encouraging environment, we as
 contributors and maintainers pledge to making participation in our project and
 our community a harassment-free experience for everyone, regardless of age, body
 size, disability, ethnicity, gender identity and expression, level of experience,
 nationality, personal appearance, race, religion, or sexual identity and
 orientation.
 
 ## Our Standards
+Habitat is a diverse community of professionals and volunteers who come from all
+over the world to make Habitat better. Community members may fulfill many roles
+including mentoring, teaching, and connecting with other members of the community.
 
-Examples of behavior that contributes to creating a positive environment
-include:
+Be careful in the words that you choose. Be kind to others. Practice empathy. Don't
+insult or put down others. Remember that sexist, racist, and other exclusionary jokes
+can be offensive to those around you. If you think your conversation is making another
+community member uncomfortable _or_ if they tell you so, stop immediately, make amends,
+and move forward.
 
+As you are working with other members of the community, please keep in mind the following guidelines, which apply equally to founders, mentors, those who submit new features and pull requests, and to anyone who is seeking help and guidance.
```

Comment:
Minor style nitpick: other lines terminate around the 80 char mark, these do not.